splice_access_api/src/main/java/com/splicemachine/access/configuration/PipelineConfiguration.java

```diff
@@ -146,8 +146,8 @@
     public static final String MAX_DEPENDENT_WRITES = "splice.client.write.maxDependentWrites";
     public static final int DEFAULT_MAX_DEPENDENT_WRITES = 40000;
 
-    public static final String IPC_THREADS="hbase.regionserver.handler.count";
-    public static final int DEFAULT_IPC_THREADS = 200;
+    public static final String IPC_THREADS="hbase.regionserver.metahandler.count";
+    public static final int DEFAULT_IPC_THREADS = 10;
```

Comment:
CDH 5.12 is back to using hbase.regionserver.handler.count threads for RPCs (default thread pool).
We should not use priority thread pool's configuration (hbase.regionserver.metahandler.count) as basis for our configuration parameters.

Reply:
Adding a couple of related prior discussions on this subject:
https://splice.atlassian.net/browse/SPLICE-1379
https://splice.atlassian.net/browse/SPLICE-1392